debian/rules

```diff
@@ -134,6 +134,14 @@ override_dh_strip:
 override_dh_shlibdeps:
```

Comment:
@dmick sorry for the misleading example, yeah, lemme copy another warning message.